web/styles/modal.css

```diff
@@ -9,10 +9,7 @@
 /* Styles for the Micromodal-based modals */
 .modal__overlay {
     position: fixed;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
+    inset: 0;
```

Comment:
This will be transpiled to `top` et al. by #24751, otherwise https://caniuse.com/mdn-css_properties_inset applies.